### ISSUE TITLE
Scrap locations

### ISSUE THREAD

**Issue by mrakrmakmrka** [Wexop/OptimalScrapsOrganization#2]
I've noticed that scrap always when using the sort function, scrap always spawns to the right of the ship exit (marked with a red arrow in the screenshot). Why is this bad ? When there is too much scrap on the ship (more than 45), even if you use a mod that increases the number of scrap available for location on the ship to 999, the scrap somehow starts to fall under the ship. And it does it from the side from which the scrap is located. It is from this side (marked with a red arrow) it is impossible to get the scrap without using cheater mods or glitch with a ladder. But, if the loot is left where on the screenshot I indicated with a yellow line, the scrap can be easily pulled out from under the ship without using anything/anyone from the side. 

![123](https://github.com/user-attachments/assets/a4200a41-f0b0-4f58-9870-3dbde5994e72)

That is, what I suggest you to do additionally, is the ability to customize the place from where the scrap starts sorting.

But anyway, Thank you for the mod, it's the best mod for sorting.

**Comment by Wexop** [comment]
Hello !

Thanks for your report :) I tested with 100+ items without any mod that increases the max items on ship, it seems to work

BUT, someone else reported an issue when you place items on locker or any where else, and this time items fall on the ground each time you use reorder commands.

Do you think that it could be the same issue ? Because i couldn't reproduce your issue 🤔 

I'll upload soon a fix for the placed items issue, can you try with this new version (1.0.3) and tell me if this solve your issue ?

Thanks again for your message :)

**Comment by mrakrmakmrka** [comment]
> 
> BUT, someone else reported an issue when you place items on locker or any where else, and this time items fall on the ground each time you use reorder commands.
> 
> Do you think that it could be the same issue ? Because i couldn't reproduce your issue 🤔


Hi!
I don't know what to write about the lockers as I don't use them.
As for my problem, I was able to reproduce it this way, I collected a lot of scrap (screenshot below), 
![1](https://github.com/user-attachments/assets/5015a497-ed52-4848-b111-bcf75b5bb527)
flew into orbit, waited for the notification in the lower right corner about the successful saving of the game, went out to the main menu and back in, then landed in the company building.
The result is this, 2 items fell under the ship, a lamp and a duckling. 
![2](https://github.com/user-attachments/assets/61617e0a-d20f-4142-9411-e3cd1580b86b)
![3](https://github.com/user-attachments/assets/52791ead-4b8b-4d33-aa15-a9f27598ba7b)

But the nature of this game behavior is unclear to me, since the items can end up under the ship to the right of the stairs (which can be used to climb the ship). I recognized the place where the duckling fell through only now, when I reproduced this situation. Where there is a lamp, the situation is standard, there can be several items there at once.
Your mod, by the way, when using the sorting function picks them up and sorts (as if pulls out from under the ship), this is a kind of solution. 
So overall, it turned out to be a non-issue. 
Thanks again for such a cool mod!
